src/modules/organization/queries.ts

```diff
@@ -8,3 +8,12 @@ export const ALL_INTERNAL_GROUP_POSITIONS = gql`
     }
   }
 `
+
+export const ALL_INTERNAL_GROUPS = gql`
+  query AllInternalGroupsQuery {
+    allInternalGroups {
+      id
+      name
+    }
+  }
+`
```

Comment:
As mentioned above, probably needs to be coordinated with #31 to avoid redundancy.